src/types/MaybeArray.ts

```diff
@@ -0,0 +1 @@
+export type MaybeArray<T> = T | Array<T>;
```

Comment:
[nit] Totally optional, but instead of creating or own type, we can use ArrayOrSingle from ts-essentials (https://github.com/ts-essentials/ts-essentials#ArrayOrSingle).

Reply:
Good call! Didn't know that was a thing